etl/models/models.go

```diff
@@ -114,7 +114,7 @@ type Request struct {
 	RecordReference          []byte `gorm:"primary_key;auto_increment:false"` // Request reference.
 	Type                     RequestType
 	CallType                 string
-	ObjectReference          []byte // Reference to the corresponding object.
+	ObjectReference          []byte `gorm:"column:object_ref"` // Reference to the corresponding object.
```

Comment:
May be better to rename field in db to `object_reference` ?

Reply:
Done